### ISSUE TITLE
Apps appearing in PS update list that shouldn't be there

### ISSUE THREAD

**Issue by iestynapmwg** [j-hc/zygisk-detach#15]
This is not a bug report per sé, as the module is correctly removing the one app from the Play Store's list of updates that i asked it to. But oddly, after installing this module, i now have 5 other apps showing up in the list of updates that have never been there before. These are apps that were installed from other sources, and while the package names match, the signatures don't, and so any attempt at updating from the Play Store fails. As it should, as i don't want to install the PS versions.

Short of adding these extra packages to the detach list, is there any way to prevent these duplicates from showing in my PS update list? I'm not sure what kind of debugging output i can provide, as the module is doing what it's supposed to. It's just that these other apps never appeared in my PS list before installing this module, as they weren't installed from the PS.

In short, the DETACHING from PS works, but it's also ATTACHING other apps that weren't before.

**Comment by j-hc** [comment]
unrelated to zygisk detach. play store does it.

**Comment by rpgod** [comment]
Nope its happening after this module is installed, now installed modded apk also being listed in ps update

**Comment by j-hc** [comment]
> Nope its happening after this module is installed, now installed modded apk also being listed in ps update

prove it. show the updates list with zygisk-detach enabled and disabled.
